sqs/src/main/scala/akka/stream/alpakka/sqs/SqsSourceSettings.scala

```diff
@@ -16,7 +16,8 @@ final class SqsSourceSettings private (
     val maxBatchSize: Int,
     val attributeNames: immutable.Seq[AttributeName],
     val messageAttributeNames: immutable.Seq[MessageAttributeName],
-    val closeOnEmptyReceive: Boolean
+    val closeOnEmptyReceive: Boolean,
+    val visibilityTimeout: Option[Int]
```

Comment:
I'd rather see this as a `FiniteDuration`. Possibly representing `None` with `Duration.Zero` instead.

Reply:
it might be a problem setting the default to `Zero`, since it's a valid value (see [doc](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html#terminating-message-visibility-timeout))
so if one does not want to use this feature, we won't override the default value in SQS queue console.